specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2022-03-01/Marketplace.json

```
@@ -1925,12 +1925,12 @@
           "description": "Indicating whether all subscriptions are selected (=true) or not (=false).",
           "type": "boolean"
         },
-        "allItemsApproved": {
+        "approveAllItems": {
           "description": "Indicating whether all items are approved for this collection (=true) or not (=false).",
           "type": "boolean",
           "readOnly": true
```

Can we add the previous name of this property here back using the x-ms-client-name so that the SDK will not break?
```suggestion
          "readOnly": true,
          "x-ms-client-name": "allItemsApproved"
```
Or at least we could add this to Go SDK's configuration, since it is not necessary to break the SDK when fixing this issue

---

OK, I take a careful look at the changelog of Go SDK, fortunately that we did not release a new SDK with the two incorrect properties, therefore this breaking (and the one below) are OK to have